src/event_listener/handlers.py

```
@@ -0,0 +1,315 @@
+
+    async def invoke_orchestrator(self, state, thread_id: str) -> Dict[str, Any]:
+        """Invoke the orchestrator with the given state.
+
+        Args:
+            state: The state object to process
+            thread_id: Unique thread ID for this event
+
+        Returns:
+            dict: Orchestrator result
+        """
+        print(f"Invoking orchestrator with state: {state}")
```

Ruff (0.11.9)

`print` found

Remove `print`

(T201)